akka-actor-typed-tests/src/test/scala/docs/akka/typed/SpawnProtocolDocSpec.scala

```diff
@@ -30,6 +31,8 @@ import akka.actor.typed.Scheduler
 
 object SpawnProtocolDocSpec {
 
+  // Silent because we want to name the unused 'context' parameter
+  @silent
```

Comment:
maybe add pattern

Reply:
Merged, added in https://github.com/akka/akka/pull/27433